src/ContentHub.php

```diff
@@ -12,6 +12,10 @@
 
 class ContentHub extends Client
 {
+    // Override VERSION inherited from GuzzleHttp::ClientInterface
+    const VERSION = '0.6.4';
```

Comment:
Any way to automate this? https://github.com/sebastianbergmann/version seems like it might help, it has 14M installs so other projects are finding it useful.